src/main/blackbox/blackbox.h

```diff
@@ -23,7 +23,10 @@ void blackboxLogEvent(FlightLogEvent event, flightLogEventData_t *data);
 
 void initBlackbox(void);
 void handleBlackbox(void);
+void startInTestMode(void);
+void stopInTestMode(void);
 void startBlackbox(void);
 void finishBlackbox(void);
+bool inMotorTestMode(void);
```

Comment:
btw I notice that it seems like things are missing and thats probably why you defined these in header files